Formula/serf.rb

```diff
@@ -4,8 +4,8 @@ class Serf < Formula
   desc "Service orchestration and management tool"
   homepage "https://serfdom.io/"
   url "https://github.com/hashicorp/serf.git",
-      :tag => "v0.7.0",
-      :revision => "0df3e3df1703f838243a7f3f12bf0b88566ade5a"
```

Comment:
@DomT4 what's the rule on blank line or no blank line between `head` and what's above it, when what's above it is a `:revision` not a `revision 1`?